src/query.ts

```diff
@@ -1,3 +1,5 @@
+import {Match} from './text-expander-element'
```

Comment:
This creates a circular import which we'd like to avoid if we can. Can you please move `Match` into a separate `src/types.ts` file?

Reply:
Removed the cyclic dependency.